src/Coxeter_triangulation/include/gudhi/Cell_complex.h

```diff
@@ -0,0 +1,376 @@
+/*    This file is part of the Gudhi Library. The Gudhi library
+ *    (Geometric Understanding in Higher Dimensions) is a generic C++
+ *    library for computational topology.
+ *
+ *    Author(s):       Siargey Kachanovich
+ *
+ *    Copyright (C) 2019 Inria
+ *
+ *    Modification(s):
+ *      - YYYY/MM Author: Description of the modification
+ */
+
+#ifndef CELL_COMPLEX_H_
+#define CELL_COMPLEX_H_
+
+#include <Eigen/Dense> 
+
+#include <algorithm>
+#include <gudhi/Permutahedral_representation/Simplex_comparator.h>
+#include <fstream> // for Hasse_diagram_persistence.h
```

Comment:
Include what you use:
```suggestion
#include <vector>
#include <map>
#include <utility>  // for std::make_pair

#include <gudhi/Permutahedral_representation/Simplex_comparator.h>
```

If `#include <fstream>` is required by Hasse_diagram_persistence.h, it is up to this file to include it.